docs/asciidoc/static/codec.asciidoc

```diff
@@ -0,0 +1,13 @@
+:register_method:	true
+:encode_method:		true
+:decode_method:		true
+:plugintype:    	codec
+:pluginclass:   	Codecs
+:pluginname:    	example
+:pluginnamecap: 	Example
+
+:getstarted: For this example, we are going to examine the example codec plugin at https://github.com/logstash-plugins/logstash-codec-example/ .
```

Comment:
I'd flip these intros around to be a bit more active: "Let's step through creating a/an {plugintype} plugin using the  https://github.com/logstash-plugins/logstash-input-example/[{plugintype} example plugin]."